src/writer/settings.ts

```diff
@@ -133,4 +140,9 @@ export async function action(body): Promise<void> {
   }
 
   if (existingSettings.avatar !== msg.payload.avatar) await clearStampCache('space', space);
+
+  if (existingSpace?.domain != msg.payload.domain) {
+    await addToWalletConnectWhitelist(msg.payload.domain);
```

Comment:
I am still wondering if we want to add all domains (any space to add it) or just turbo spaces

Reply:
We need to decide whether we allow `domain` update for all spaces, or just turbo

Reply:
If that's simpler we can allow only for turbo and the ones who already had a domain set to update domain.

Reply:
We need to set boundaries on the definition of "already had a domain".

For now, if a non-turbo space has already set a domain, he can change it.

But what happen when a non-turbo space currently has a domain, but remove it (by error, etc ...) ? The way it work now, the space lose the ability to put it back. To enable this, we need a way to mark these spaces as custom domain enabled

Reply:
All i worry about this PR is that someone may add many unwanted domains to walletconnect as any space can add a domain

Reply:
Otherwise looks good

Reply:
> All i worry about this PR is that someone may add many unwanted domains to walletconnect as any space can add a domain

Working on a PR to prevent that. domain edition will be limited to turbo, and spaces with existing domains only

Reply:
See https://github.com/snapshot-labs/snapshot-sequencer/pull/488

This PR can be merged after the other one